src/platform/site-wide/user-nav/components/DropDownPanel/DropDownPanel.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: DropDownPanel
+name: DropDownPanel
+tags: panel, component
+---
+
+import DropDownPanel from './DropDownPanel'
+
+### Code:
+```javascript
+import DropDownPanel from '@department-of-veterans-affairs/component-library/DropDownPanel'
+
+<div style={{ backgroundColor: '#112e51' }}>
+  <DropDownPanel
+    buttonText='Helpdesk'
+    cssClass='va-dropdown'
+    isOpen={this.state.open}
+    contents='Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce ullamcorper at eros eu suscipit. Ut imperdiet libero et luctus pretium.'
+    clickHandler={() => this.setState({ open: !this.state.open })}>
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Fusce ullamcorper at eros eu suscipit. Ut imperdiet libero et luctus pretium.'
+  </DropDownPanel>
+</div>
+```
+
+### Rendered Component
+export class RenderComponent extends React.Component {
```

Comment:
This is missing the wrapping backticks.

Reply:
@randimays I don't think these files are intended for use as basic browser-based docs. I think (I don't know) that they were created for use with some other documentation tool, possibly one that predated the use of Storybook by the Design System team (again, I'm just guessing). The portion of the file that _is_ within backticks appears to be the code sample, whereas the rest of the file appears to be an attempt to actually render the component as described in the code sample. Because I don't know how these are (or were) being used, I can't do anything more than guess.

@ataker Do we know what the purpose of these mdx files is? Do we need to copy this over when we port this from `component-library` to `vets-website`?